codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -131,6 +164,18 @@ protected Result beginEncode(HttpResponse headers, String acceptEncoding) throws
                         wrapper, compressionLevel, windowBits, memLevel)));
     }
 
+    /**
+     * For customization should implement this method that
+     * performs the necessary business logic.
```

Comment:
@teaey please reword to:

Returns {@code true} if the compression should be used. May be override to change default behaviour